src/beacon.js

```diff
@@ -19,6 +21,18 @@ function fallbackBeacon(url, data, sync) {
   return true;
 }
 
+const getMaxLength = function() {
+  if (typeof window === 'undefined') {
+    return DEFAULT_MAX_MESSAGE_SIZE;
+  }
+
+  if (window.navigator && window.navigator.userAgent && window.navigator.userAgent.indexOf("Edg") > -1) {
```

Comment:
Edg? I imagine it still works, but it's odd.

Reply:
https://learn.microsoft.com/en-us/microsoft-edge/web-platform/user-agent-guidance

they seem like they're recommending using the user agent client hints instead of the string but if we're using the string then `Edg` is definitely the right thing to match.